xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/autosave.js

```diff
@@ -96,9 +96,9 @@ editors.AutoSave = Class.create({
     // Labels
     var autosaveLabel = new Element('label', {'class': 'autosave', 'for' : "doAutosave"});
     autosaveLabel.appendChild(this.autosaveCheckbox);
-    autosaveLabel.appendChild(document.createTextNode(" Autosave"));
+    autosaveLabel.appendChild(document.createTextNode("$services.localization.render('core.edit.autosave')"));
```

Comment:
There was a space at the beginning of the `" Autosave"` string. Was that needed before?

Reply:
Indeed I missed this one. I can add it actually. It might be nicer on the UI.